packages/status-react/src/utils/createMessenger.ts

```diff
@@ -1,22 +1,22 @@
 import { Messenger } from '@status-im/core'
-import { getNodesFromHostedJson } from 'js-waku'
+import { getPredefinedBootstrapNodes } from 'js-waku'
+import { Fleet } from 'js-waku/build/main/lib/discovery/predefined'
 import { Protocols } from 'js-waku/build/main/lib/waku'
 
 import type { Identity } from '@status-im/core'
+import type { CreateOptions } from 'js-waku/build/main/lib/waku'
 
-function createWakuOptions(env: string) {
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  let bootstrap: any = { default: true }
+function createWakuOptions(env: string): CreateOptions {
+  let bootstrap: CreateOptions['bootstrap'] = {
+    default: true,
+  }
 
   if (env === 'test') {
     bootstrap = {
-      getPeers: getNodesFromHostedJson.bind({}, [
-        'fleets',
-        'wakuv2.test',
-        'waku-websocket',
-      ]),
+      getPeers: getPredefinedBootstrapNodes(Fleet.Test),
```

Comment:
@D4nte can you check, whether this setup is correct?

Reply:
`getPeers` takes a function. You can either do

```ts
 bootstrap: {
        getPeers: getPredefinedBootstrapNodes.bind({}, Fleet.Test).map(Promise.resolve),
      }
```

or 

```ts
bootstrap: {
        peers: getPredefinedBootstrapNodes(Fleet.Test).map(a => a.toString())
      },
```